### ISSUE TITLE
stopped working

### ISSUE THREAD

**Issue by sakshamgoyal01** [AllToMP3/alltomp3-app#182]
Windows 10 64bit, version 0.3.17, URL: https://open.spotify.com/playlist/24K0oTqQy5gGyerJv9Mrza

As of 2 days ago, it was working with minor issues. It works for the first 5-10 minutes (downloads about 10-40 songs) and then app goes full white and stops working (does not show as not responding... just a white app with no activity). But today it is stuck at 0 songs downloaded but the app going white does not happen anymore. I have restarted the app multiple times but nothing is happening.

**Comment by braddale** [comment]
sakshamgoyal01  have you managed to find a fix yet? Mine has also been down for 4 days now.

**Comment by sakshamgoyal01** [comment]
I haven't tried to fix it because idk how to...
I just tried it again and it still won't work for me.

**Comment by Darkflare14** [comment]
@NTag Do you still support this application? It hasn't been working since June.

**Comment by braddale** [comment]
I downloaded it on anther laptop and it worked perfectly for the day.
Next day it wont work. Looks like Spotify prevents the programme from
downloading after 1 day.

Kind Regards

Bradley Dale
1/271 Kohimarama Road
St Heliers
Auckland
Tel: 027 522 6856

**Comment by NTag** [comment]
I released an update (0.3.18), it may help 🤞

**Comment by sakshamgoyal01** [comment]
well immediately it seems like its running much better. no issues right now. (i manually uninstalled the old version and then installed the new one)

**Comment by Darkflare14** [comment]
Great! Thank you so much for updating this.

One more issue I was curious about is the memory leak. It turns the GUI white after sometime and takes up all of the memory of the computer if downloading a large playlist. Any solutions?

________________________________

**Comment by braddale** [comment]
Yes Im also battling at the moment. Downloads 6 tracks then dead. Install,
uninstall no luck lately.


Kind Regards

Bradley Dale
1/271 Kohimarama Road
St Heliers
Auckland
Tel: 027 522 6856

**Comment by m-p-x** [comment]
-> braddale 
I've never been able to DL a complete album but missing tracks has been successfully DL'd individually :)
so u might try to insert explicit tracks/artist 
for now, nothing works for me, 
- no spotify-link
- no Youtube-link
- no explicit search 
:(((

**Comment by m-p-x** [comment]
Tried a new uninstall/reinstall today, 
still no luck, totally dead :(

It's a pity, this little gem was really nice when it worked ...

**Comment by ian02054** [comment]
Same here.. was working to download a couple of playlists, then app window went white, and now nothing working.. 
tried uninstall /re-install.. no luck. Tried deezer & Spotify, albums, playlists, single tracks

**Comment by m-p-x** [comment]
hmm, weird, really weird ...
A couple of days ago it worked again, 
but,
after about two days it died :(
tried a new uninstall/reinstall but no response :(

**Comment by icedcrackerz** [comment]
Still not working :(

**Comment by m-p-x** [comment]
today it worked again for me :)

**Comment by AlexWhitehouse** [comment]
Anyone having the issue of the white screen and it apparently freezing READ THIS...

It appears to have frozen but if you check the directory you have set All2MP3 to download to it is in fact actually s, still downloading them, albeit without the GUI working. This appears to only happen when you try to download large playlists. Watch in the directory you have set the program to save songs to and it will continue to populate the rest of the album. Patience is key!!

**Comment by m-p-x** [comment]
never experienced "white screen", 
my problem is that it's only is working for a couple of days, then it doesn't DL anything, finds what I'm looking for but nothing DL'ed ..., waiting some days and it starts working again ??? 
I've installed it on another PC and for now it works there but it's stopped (again) on my main PC ??? 
weird, weird, weird ...

**Comment by NTag** [comment]
I released an update. However it only supports YouTube URLs, Spotify/Deezer/song title searches are broken for now, sorry.

**Comment by ian02054** [comment]
Hi Basile, Should this work for youtube playlists as well? I tried a couple, and they didn't work.. (https://www.youtube.com/playlist?list=PLFF3CFBD9770F8C45 & https://www.youtube.com/playlist?list=PLm7YbuiZ9Akqlny9mO5AG5tq_jAIF4yYo)

**Comment by NTag** [comment]
No, I think they are broken too. Only simple video URLs should work.

**Comment by ian02054** [comment]
Great, thanks for confirmig...appreciate all the effort.
